build/browserstack-legacy-1.json

```diff
@@ -0,0 +1,40 @@
+{
+	"username": "BROWSERSTACK_USERNAME",
+	"key": "BROWSERSTACK_KEY",
+	"test_framework": "qunit",
+	"test_path": [
+		"test/index.html",
+		"test/logs.html"
+	],
+	"browsers": [
+		"chrome_previous",
+		"firefox_previous",
+		"opera_previous",
+		"safari_5_1",
+		{
+			"browser": "safari",
+			"browser_version": "6.0"
+		},
+		{
+			"browser": "safari",
+			"browser_version": "6.2"
+		},
+		{
+			"browser": "safari",
+			"browser_version": "8.0"
```

Comment:
@mzgol I hope I got it now.

I removed Safari `previous` and changed 7.1 to 8.0.